debian/mariadb-server-10.5.postrm

```diff
@@ -74,7 +82,7 @@ if [ "$1" = "purge" ] && [ -f "/var/lib/mysql/debian-$MAJOR_VER.flag" ]; then
       # /var/lib/mysql is a mount point
       rmdir --ignore-fail-on-non-empty /var/lib/mysql || true
     fi
-    rm -rf /var/run/mysqld # this directory is created by the init script, don't leave behind
+    rm -rf /run/mysqld # this directory is created by the init script, don't leave behind
```

Comment:
@robertbindar when testing this, check that this /run/mysqld is actually the correct path and we didn't forget any /var/run/mysqld in any config file.